src/sentry/rules/conditions/event_frequency.py

```diff
@@ -533,6 +535,207 @@ def get_preview_aggregate(self) -> tuple[str, str]:
         return "uniq", "user"
 
 
+class EventUniqueUserFrequencyConditionWithConditions(EventUniqueUserFrequencyCondition):
+    id = "sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyConditionWithConditions"
+
+    def query_hook(
```

Comment:
i'm not quite certain how query_hook is used, seems like in the normal delayed flow we use `batch_query_hook`?

Reply:
got it! this should still work in any case

Reply:
💯 - to what dan said. Sorry I had that comment in a pending message 😅